x-pack/plugin/shutdown/src/internalClusterTest/java/org/elasticsearch/xpack/shutdown/NodeShutdownShardsIT.java

```diff
@@ -137,6 +145,70 @@ public void testShardStatusIsCompleteOnNonDataNodes() throws Exception {
         assertThat(getResp.getShutdownStatuses().get(0).migrationStatus().getStatus(), equalTo(COMPLETE));
     }
 
+    // GWB> RENAME THIS TEST AND ALSO ADD JAVADOC
```

Comment:
I think this still needs to be resolved?

Reply:
Thanks!